make_helpers/defaults.mk

```diff
@@ -128,6 +128,9 @@ TRUSTED_BOARD_BOOT		:= 0
 # Build option to choose whether Trusted firmware uses Coherent memory or not.
 USE_COHERENT_MEM		:= 1
 
+# Use tbbr_oid.h instead of platform_oid.h
+USE_TBBR_DEFS			:= 0
+
```

Comment:
Unfortunately, I found the alternative code did not work with `$(call assert_boolean, ...)`

So, I moved the ERROR_DEPRECATED check to the top Makefile  (after including platform.mk).